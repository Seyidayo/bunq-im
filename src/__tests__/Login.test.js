import renderer from 'react-test-renderer';
import LoginPage from 'app/components/pages/Login.page';

describe('<LoginPage />', () => {
  it('It renders correctly and matches snapshot', () => {
    const tree = renderer.create(<LoginPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
