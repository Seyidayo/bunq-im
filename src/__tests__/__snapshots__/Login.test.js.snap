// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> It renders correctly and matches snapshot 1`] = `
<form
  className="container login__container"
  id="login-form"
  onSubmit={[Function]}
>
  <h1
    className="h4"
  >
    bunqIM
  </h1>
  <label
    hidden={true}
    htmlFor="email-input"
  >
    Email address
  </label>
  <input
    id="email-input"
    name="email"
    onChange={[Function]}
    placeholder="Email address"
    type="email"
    value="bunqer@bunq.com"
  />
  <label
    hidden={true}
    htmlFor="password-input"
  >
    Password
  </label>
  <input
    id="password-input"
    name="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value="bunqer123"
  />
  <button
    disabled={false}
    type="submit"
  >
     Â  Log in
  </button>
</form>
`;
